auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -16,7 +16,7 @@
           <v-btn
             id="add-name-request-btn"
             class="font-weight-regular"
-            color="primary"
+            color="#1669BB"
```

Comment:
Can we move it to scoped scss at the bottom ?

Reply:
Absolutely, i could also define this blue (app-blue) on the vue instance, as primary === app-blue in our approach to keeping things consistent across all the apps.

Reply:
Yes, please define a Vuetify theme colour as needed.

Reply:
Updated in next commit 👍